paparazzi/build.gradle

```diff
@@ -53,6 +53,10 @@ def generateTestConfig = tasks.register("generateTestConfig") {
       writer.writeLine("app.cash.paparazzi")
       writer.writeLine(".")
       writer.writeLine("28")
+      def androidSdkFolder = System.getenv("ANDROID_SDK_ROOT")
```

Comment:
Since the project here does not have the Android plugin, I'm using the env variable. I think it should work, but it might not on some machines. Another option would be to try and read the value from `local.properties` file - I think AGP is automatically writing the path to that file.

Reply:
makes sense, as is.